forge-gui/res/cardsfolder/upcoming/travel_through_caradhras.txt

```diff
@@ -0,0 +1,9 @@
+Name:Travel Through Caradhras
+ManaCost:5 G
+Types:Sorcery
+A:SP$ Vote | Defined$ Player | StoreVoteNum$ True | VoteType$ RedhornPass,MinesofMoria | SubAbility$ VoteRedhornPass | SpellDescription$ Council's dilemma — Starting with you, each player votes for Redhorn Pass or Mines of Moria. For each Redhorn Pass vote, search your library for a basic land card and put it onto the battlefield tapped. If you search your library this way, shuffle. For each Mines of Moria vote, return a card from your graveyard to your hand.
```

Comment:
for UI could you try `VoteType$ Redhorn Pass,Mines of Moria`
You will need to similarly space VoteNumRedhorn Pass
But I think it should work?
No spaces needed in SubAbility names!

Reply:
huh this acually works. Thanks